nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -433,7 +433,9 @@ def run(self, dryrun=False) -> List[uuid.UUID]:
 
         raw_notification_ids = []
         for notification in notifications:
-            self.logger.info(f"Processing notification `{notification.subject}`.", extra={"object": notification})
+            self.logger.info(
+                f"Processing notification `{notification.subject}` with ID `{notification.msg_id}` from source `{notification.source.name}`."
+            )
```

Comment:
This change removes the link to the `notification` object so that it's no longer possible to click through from the log message to the notification itself. Is that desirable?

Reply:
the Notification object is not a Nautobto object, so it's not linked and only a partial representation is shown.